### PR TITLE
How do we determine what interaction networks are feasible?

### PR DESCRIPTION
**Pull request by TanyaS08**

### CODE REVIEW

**Comment by TanyaS08** [comment]
A long term 'food for thought' comment - should we maybe rethink the wording of the question. I don't know if it is because it is in the ML section that I read this Q as 'are the interaction networks we predicted biologically feasible' but at the moment this section reads more as about the long term feasibility/stability of networks. Although maybe the two are linked in a way... 🧐

File: manuscript.md [hunk 1]
```diff
@@ -30,6 +34,22 @@
...
+These typically rely on _network embeddings_---where each node (species) in the network is assigned a value in a latent space, and the resulting network topology is generated stochastically based on properties of the position of nodes in that latent space. Generative network models have long used allometry as a single-dimensional latent space---naturally we want to extend this to traits in general [@Allesina2008GenMod].
+
+The second approach to understand stability is through _dynamics_.
+Early models of community dynamics (Lotka-Volterra, MacArthurs Consumer-Resource Model) rely on the assumption of linear interaction effects. However, models of bioenergetic community dynamics have shown promise in basing our understanding of dynamics in food-webs, where the functional response of one species on another is grounded in the understood relationship between allometry and metabolism [@Delmas2017SimBio].
+
+An additional consideration is that "stability" and "feasibility" have different dimensions---resilience to environmental change, extinction, etc [@Dominguez-Garcia2019UnvDim].
```
Comment (tpoisot) [inline manuscript.md:50]: develop this a little bit more

**Review by tpoisot — CHANGES_REQUESTED** [review]

File: manuscript.md (hunk 1 above)
Comment (gottacatchenall) [reply manuscript.md]: i think Tanya's later commit resolves this

**Review by gottacatchenall — APPROVED** [review]

File: manuscript.md [hunk 2]
```diff
@@ -30,6 +34,48 @@
...
+understood relationship between allometry and metabolism
+[@Delmas2017SimBio].
+
+An additional consideration is the multidimensional nature of "stability"
+and "feasibility" e.g resilience to environmental change vs extinctions
+[@Dominguez-Garcia2019UnvDim] and how these different disturbances will
+propagate along the various level of organisations within a network
+[@Kefi2019AdvOur], as well as their effect over space such as considering
+metacommunities and the affect of dispersal/linkages between them in 
+dampening these effects e.g building on the work of @Gravel2016StaCom.
```
Comment (DominiqueCaron) [inline manuscript.md:76]: See, here you are constrasting stability and feasibility, but the difference between the two is not clear in the previous paragraphs

**Review by tpoisot — APPROVED** [review]

**Pull request merged**